.dockerignore

```diff
@@ -3,6 +3,5 @@ node_modules/
 dist/
 build/
 .github/
-.env
```

Comment:
Do we need to remove the `.env` file from the docker ignore? If we're registering it in the backend then I don't think it needs to get built into the image.